### ISSUE TITLE
profile photo bug

### ISSUE THREAD

**Issue by jdaudier** [loicginoux/joanne_project#51]
Profile photo is being weird when I try to upload a photo (I am using the latest Chrome browser)

http://www.evernote.com/shard/s23/sh/bbd206aa-0f88-4728-96c0-906a3b40ab20/516698650ef89d9e004f8b47ad284eb6

http://www.evernote.com/shard/s23/sh/1b91d85e-92a4-4516-bb65-29c6a9a565f2/e9a3ac7dde81c443e01d97a84b07503c

**Comment by jdaudier** [comment]
When you choose a photo to upload, did you notice that it squashes it before you press UPDATE?

**Comment by loicginoux** [comment]
yes I noticed it but I don't think there is an easy solution for this. I explain:
when you first add a file before updating or uploading, it displays the original image from the local machine, it is not cropped yet, and I made it fit in the box, therefore some image can appear squashed if they are very long or very large. After updating, the server crop the image from the center with the right dimensions so it doesn't appear squashed anymore.
To fix the issue, I would need to simulate the crop made on the server and I am not sure it is that easy to do. I might have an idea that I can investigate but I am not sure it will work
